### PR TITLE
webpack build system for splunk app

### PR DESCRIPTION
**Pull request by anthonythleung**
This PR adds webpack build system for the Splunk app.

`npm run build` in `app/onepassword_events_api/` builds the web app in development mode.

`npm run build-release` in `app/onepassword_events_api/` and build the web app in production mode.

A new script `util/build_version.js` is added and automatically run to sync all the version numbers and build hash for all files, including `app.conf`, with ~~`package.json`~~ `Makefile` as the source of truth.

These files are synced with `util/build_version.js`:

* `app.js`'s title: `"1Password Events API for Splunk Setup Page - Version [version]"`
* `app.conf`'s version and build
* `setup_page_dashboard.xml `'s js script name: `script="build/main_[contenthash].js`

Minimal changes to the actual app is done in this PR to keep this PR small and ensure nothing breaks.

### CODE REVIEW

File: app/onepassword_events_api/default/data/ui/views/setup_page_dashboard.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@ http://docs.splunk.com/Documentation/Splunk/latest/Viz/PanelreferenceforSimplifi
 -->
 <dashboard isDashboard="false"
            hideTitle="true"
-           script="javascript/setup_page_130.js"
+           script="build/main_92876d62430c7f26b4f3.js"
            stylesheet="styles/setup_page.css">
```
Comment (hilaryhacksel) [inline app/onepassword_events_api/default/data/ui/views/setup_page_dashboard.xml:11]: We'll need to also add the hash here so the css isn't cached.

**Review by hilaryhacksel — COMMENTED** [review]
Looks good! Just one small thing and I'll give it a test now

**Review by hilaryhacksel — APPROVED** [review]
Tested and works well!

**Review by meek — APPROVED** [review]
Looks good! Tested and works well.

**Pull request merged**